tests_ollm/endpoints.json

```diff
@@ -2,71 +2,86 @@
   "dev": [
     {
       "url": "https://text.customer-endpoints.nimbus.octoml.ai",
-      "model": "codellama-7b-instruct-fp16"
+      "model": "codellama-7b-instruct-fp16",
+      "context_size": 4096
     },
     {
       "url": "https://text.customer-endpoints.nimbus.octoml.ai",
-      "model": "codellama-13b-instruct-fp16"
+      "model": "codellama-13b-instruct-fp16",
+      "context_size": 4096
     },
     {
       "url": "https://text.customer-endpoints.nimbus.octoml.ai",
-      "model": "codellama-34b-instruct-int4"
+      "model": "codellama-34b-instruct-int4",
+      "context_size": 4096
     },
     {
       "url": "https://text.customer-endpoints.nimbus.octoml.ai",
-      "model": "codellama-34b-instruct-fp16"
+      "model": "codellama-34b-instruct-fp16",
+      "context_size": 16384
```

Comment:
Looks like just now int4 model is used instead of fp16 in dev and context_size for it is 4096

Reply:
Ok, changed it to 4096